### ISSUE TITLE
Clarify the Checks gibberish in builds

### ISSUE THREAD

**Issue by stefanalund** [spotify/backstage#593]
These checks makes no sense to me and should be clarified to become useful:

<img width="732" alt="Screenshot 2020-04-18 at 17 33 27" src="https://user-images.githubusercontent.com/190856/79641929-cba8b780-819a-11ea-8129-632788fb7064.png">

**Comment by OrkoHunter** [comment]
I think only the "deploy-preview" check is useful. Here is how to remove the rest
https://github.com/netlify/cli/issues/155

> ... you can remove those checks from your site's dashboard.
Go to Settings -> Build & Deploy -> and scroll down to "Outgoing Notifications". There, find the ones that say "Add rich deploy details to git commits when deploy starts/succeeds/fails", click on the 3 dots next to them, and you'll be able to remove them.

**Comment by OrkoHunter** [comment]
IMO the Netlify project name `goofy-bose` can be changed to `spotify-backstage-storybook` or similar, for their URLs to have more meaning.

**Comment by OrkoHunter** [comment]
Hi @nikek, did you get a chance to check with Netlify on this? 🙇 🙏

**Comment by nikek** [comment]
@OrkoHunter I cleaned up all the Netlify stuff as we ran out of quota. Haven't looked into setting up a payed deployment or using the open source pricing plan. It was just a hack trial on my part and we might want to look at a different solution like ZEIT.. I mean Vercel(?), or some other continuous QA hosting solution. Unless someone wants to look into it, I can pick it up next week.

**Comment by nikek** [comment]
I'll close this issue as the gibberish is gone.

**Comment by OrkoHunter** [comment]
Thank you @nikek ! 🙏
